src/app/components/work-item-new-detail/work-item-new-detail.component.ts

```diff
@@ -165,7 +184,7 @@ export class WorkItemNewDetailComponent implements OnInit, OnDestroy {
           this.loadingIteration = true;
           this.loadingArea = true;
         })
-        .switchMap(() => this.workItemService.getWorkItemById(id, owner, space))
+        .switchMap(() => this.workItemService.getWorkItemByNumber(id, owner, space))
         .do(workItem => {
```

Comment:
On the PR snapshot - this still fails. The URL displays the system number but GET call fails to load all the details.

Reply:
I have tried the second snapshot and worked fine for me.